pom.xml

```diff
@@ -28,19 +36,18 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
-            <groupId>com.epam.db</groupId>
-            <artifactId>DariaBelozertseva</artifactId>
-            <version>1.0</version>
-            <scope>system</scope>
-            <systemPath>${project.basedir}/src/test/resources/hw1/calculator-1.0.jar</systemPath>
-        </dependency>
 
         <dependency>
             <groupId>io.github.bonigarcia</groupId>
             <artifactId>webdrivermanager</artifactId>
             <version>${webdrivermanager.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.1.0</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
It is still 2 dependencies to `testng`

Reply:
Oh.. It seems when I had return commit with hw1, this dependency came back again.. :(